Yafc.Model/Model/ProductionTable.cs

```diff
@@ -229,19 +223,19 @@ public class ProductionTable : ProjectPageContents, IComparer<ProductionTableFlo
         }
 
         public override async Task<string> Solve(ProjectPage page) {
-            var solver = DataUtils.CreateSolver("ProductionTableSolver");
-            var objective = solver.Objective();
+            var productionTableSolver = DataUtils.CreateSolver();
```

Comment:
Better refactoring would be to change this line to `using var productionTableSolver = DataUtils.CreateSolver();` and get rid of the Dispose calls on line 413, 449 etc.

Reply:
Thank you for the tip! Addressed in https://github.com/have-fun-was-taken/yafc-ce/pull/116/commits/67570029b81c76cdf462239f0d28e5da37a7d5cd